codes/quantum/qubits/subsystem/subsystem_sparse.yml

```diff
@@ -0,0 +1,37 @@
+code_id: subsystem_sparse
+physical: qubits
+logical: qubits
+
+
+name: 'Sparse quantum codes from quantum circuits'
+introduced: '\cite{doi:10.1145/2746539.2746608}'
+
+
+description: 'It is a constructive map from an arbitrary stabilizer code into a new subsystem code, with the same distance and number of encoded qubits. All genertors have constant weight, at the cost of additional ancilla qubits. The construction involvs the quantum error detecting circuit of the original code. Given such a circuit, there is a qubit corresponding to every output and input of each circuit element, and a set of gauge generators corresponding to each circuit element. Further, the new code can be made spatially local at the cost of more ancilla qubits.'
```

Comment:
Check spelling. Also, if there is anything quantitative that can be fished out of the paper, it would be great to do so. This is a good qualitative description BTW. "spatially local" --> "geometrically local"